core.spec.md

```diff
@@ -380,15 +424,6 @@ Schema documents MUST reference a feature version which supports all the schema
 If a processor chooses to activate support for a feature, the processor MUST activate an implementation which can [satisfy](#sec-Satisfaction) the version required by the document.
 
 
-# Extensibility
```

Comment:
I feel a little bad about entirely removing this but I think we're right that we should wait until we have a use case for keeping this. If we do put this back we'll want to change the bootstrapping algorithm to check that the argument list and location lists have subset relationships rather than equality, etc.